src/api/users/jest.config.e2e.js

```diff
@@ -4,6 +4,9 @@ require('dotenv').config({
   path: path.join(__dirname, '../../../config/env.development'),
 });
 
+// override to pass e2e tests
+process.env.FIRESTORE_EMULATOR_HOST = 'localhost:8088';
```

Comment:
I'm not particularly happy about this but you can see in `[3aa1840](https://github.com/Seneca-CDOT/telescope/runs/2210533997)` that the tests don't pass without this line. I'm assuming it's something with how the tests are running under the hood, but I'm not entirely sure. Any alternative solutions or comments are welcome as always.

Reply:
I think you're mixing firebase and localhost